dev-tools/ecs-migration.yml

```diff
@@ -1229,6 +1229,33 @@
   alias: false
   beat: metricbeat
 
+### Kibana
+
+- from: kibana.stats.uuid
+  to: service.id
+  alias: true
+  beat: metricbeat
+
+- from: kibana.status.uuid
+  to: service.id
+  alias: true
+  beat: metricbeat
+
+- from: logstash.node.host
+  to: host.hostname
```

Comment:
I also have second thoughts on this one @webmat 

Should we go for `service.hostname` for now? Or just leave it?